packages/theme/webpack.config.js

```diff
@@ -1,10 +1,13 @@
 const path = require('path');
 const webpack = require('webpack');
-const autoprefixer = require('autoprefixer');
-const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
 
-module.exports = () => {
+module.exports = (env, argv) => {
+	const isDev = argv.mode === 'development';
 	return {
+		mode: isDev ? 'development' : 'production',
```

Comment:
No need for this according to documentation https://webpack.js.org/configuration/mode/#usage

Reply:
It can be "none", but yes I can change the package scripts instead